lib/active_fedora/relation/finder_methods.rb

```diff
@@ -264,10 +264,10 @@ def condition_to_clauses(key, value)
         if value.empty?
           "-#{key}:['' TO *]"
         elsif value.is_a? Array
-          value.map { |val| "#{key}:#{RSolr.escape(val)}" }
+          value.map { |val| "#{key}:#{SolrQueryBuilder.solr_escape(val)}" }
```

Comment:
It seems that this is the only place that solr_escape is needed (after making my other suggested changes). Can we make it private?